CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 ### Changed
 
+- Revert using Github artifacts [#3342](https://github.com/tuist/tuist/pull/3342) by [@fortmarek](https://github.com/fortmarek)
 - Get the latest available version from GitHub releases instead of the Google Cloud Storage bucket [#3335](https://github.com/tuist/tuist/pull/3335) by [@pepibumur](https://github.com/pepibumur).
```

Comment:
Shouldn't we just remove this changelog entry, since nothing is changed from last release?

Reply:
there's a bunch of changes that I did not revert - `tuist update` and such will still use Github API. I think the main issue is on CI where teams use the install script